<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CGTVideoControls</class>
 <widget class="QWidget" name="CGTVideoControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CGTVideoControls</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_backwardButton">
       <property name="toolTip">
        <string>Play video backwards.</string>
       </property>
       <property name="text">
        <string>Play Backward</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_pauseButton">
       <property name="toolTip">
        <string>Pause play back.</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_forwardButton">
       <property name="toolTip">
        <string>Play video forwards.</string>
       </property>
       <property name="text">
        <string>Play Forward</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_firstFrameButton">
       <property name="toolTip">
        <string>Jump to first frame of video.</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_stepDownButton">
       <property name="toolTip">
        <string>Reverse one frame.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="_frameSlider">
       <property name="toolTip">
        <string>Jump to frame.</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBothSides</enum>
       </property>
       <property name="tickInterval">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_stepUpButton">
       <property name="toolTip">
        <string>Advance one frame.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoRepeatInterval">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_lastFrameButton">
       <property name="toolTip">
        <string>Jump to last frame of video.</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Zoom</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="_zoomSpinBox">
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="text">
        <string>GoTo Frame</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="_gotoSpinBox">
       <property name="toolTip">
        <string>Frame number for jump.</string>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_goToButton">
       <property name="toolTip">
        <string>Jump to frame nuber.</string>
       </property>
       <property name="text">
        <string>GO</string>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>28</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>_zoomSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CGTVideoControls</receiver>
   <slot>zoom_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_frameSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>CGTVideoControls</receiver>
   <slot>slider_released()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_stepUpButton</sender>
   <signal>clicked()</signal>
   <receiver>CGTVideoControls</receiver>
   <slot>step_up()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_stepDownButton</sender>
   <signal>clicked()</signal>
   <receiver>CGTVideoControls</receiver>
   <slot>step_down()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_firstFrameButton</sender>
   <signal>clicked()</signal>
   <receiver>CGTVideoControls</receiver>
   <slot>first_frame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_lastFrameButton</sender>
   <signal>clicked()</signal>
   <receiver>CGTVideoControls</receiver>
   <slot>last_frame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>474</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_goToButton</sender>
   <signal>clicked()</signal>
   <receiver>CGTVideoControls</receiver>
   <slot>go_to_frame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_backwardButton</sender>
   <signal>clicked()</signal>
   <receiver>CGTVideoControls</receiver>
   <slot>play_backwards()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_forwardButton</sender>
   <signal>clicked()</signal>
   <receiver>CGTVideoControls</receiver>
   <slot>play_forwards()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_pauseButton</sender>
   <signal>clicked()</signal>
   <receiver>CGTVideoControls</receiver>
   <slot>play_pause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_gotoSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>CGTVideoControls</receiver>
   <slot>go_to_frame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
